prow/plugins/config.go

```diff
@@ -482,6 +483,11 @@ type ConfigUpdater struct {
 	GZIP bool `json:"gzip"`
 }
 
+// CLAConfig contains the list of available context names
+type CLAConfig struct {
+	CLAContextNames []string `json:"cla_context_names,omitempty"`
```

Comment:
Without defaulting logic this is a breaking change for all existing users. We need to default `CLAContextNames` to `[]string{"cla/linuxfoundation"}` if it is not specified. (Since the empty slice would never be a reasonable config we can just check if the slice is empty to determine if we should default.)
Please add defaulting logic to this function: https://github.com/kubernetes/test-infra/blob/13e6fe9adce3cc1929cd96ad5816f20c79306b5b/prow/plugins/config.go#L947-L994